monitoring/collectors/jenkins_collector_adapter.py

```diff
@@ -42,7 +43,7 @@ def _convert_raw_stats_to_build_stats(self, raw_stats):
             build_url = self.url,
             build_id = raw_stats['id'],
             finished_at_secs = timestamp,
-            status = raw_stats['status'],
+            status = self.job_status,
```

Comment:
Here the status will be set to `SUCCESS`/`FAILURE` correctly, but the status of last stage (line 37 in this file) will be `IN_PROGRESS`